tests/test_rfc5958.py

```diff
@@ -69,7 +69,7 @@ def testOpenTypes(self):
         oneKey = asn1Object['content'][0]
         assert oneKey['privateKeyAlgorithm']['algorithm'] == rfc8410.id_Ed25519
         pkcs_9_at_friendlyName = univ.ObjectIdentifier('1.2.840.113549.1.9.9.20')
-        assert oneKey['attributes'][0]['type'] == pkcs_9_at_friendlyName
+        assert oneKey['attributes'][0]['attrType'] == pkcs_9_at_friendlyName
```

Comment:
Can we can treat exposing 'type' instead of 'attrType' as a bug in rfc5958 that has been fixed by this patch? I'd add a changeling entry noting that then.